code/campaign/src/main/java/com/adobe/marketing/mobile/campaign/FullScreenMessage.java

```diff
@@ -378,14 +377,29 @@ public void onShow(final FullscreenMessage message) {
          * <p>
          * Triggers a call to parent method {@link CampaignMessage#viewed()}.
          *
-         * @param message the {@link FullscreenMessage} being dismissed
+         * @param presentable the {@link Presentable<InAppMessage>} being dismissed
          */
         @Override
-        public void onDismiss(final FullscreenMessage message) {
+        public void onDismiss(final @NonNull Presentable<InAppMessage> presentable) {
             Log.debug(CampaignConstants.LOG_TAG, SELF_TAG, "Fullscreen on dismiss callback received.");
             viewed();
         }
 
+        @Override
+        public void onError(final @NonNull Presentable<InAppMessage> presentable, final @NonNull PresentationError presentationError) {
+            Log.debug(CampaignConstants.LOG_TAG, SELF_TAG, "onShowFailure -  Fullscreen message failed to show.");
```

Comment:
we should log the presentationError here as well if possible

Reply:
Same comment as above